### DISCUSSION TITLE
Clustering - All sites run the job immediately

### DISCUSSION THREAD

**Discussion by s3YwCf2ZbfJG4SHAfjQMAjtsf** [quartznet/quartznet#1785] [Q&A]
When my sites startup, they immediately run the job in question.   I thought commenting out "StartNow()" would prevent that from happening.

How can I configure the jobs and then only let them run based on my Trigger? 

(Clustering works great after that, only one server runs the job based on (1 per minute configuration).   One server is doing it on the odd minutes while the other is doing it on the even minutes)

```
              IJobDetail job = JobBuilder.Create<Job>()
                    .WithIdentity(Job.JobName, JobGroup)
                    .WithDescription(Job.JobName)
                    .DisallowConcurrentExecution(true)
                    .Build();

                // Trigger the job to repeat every based on Setting minutes
                ITrigger trigger = TriggerBuilder.Create()
                    .WithIdentity(Job.JobName, JobGroup)
                    //.StartNow()
                    .WithDescription("Executing JobNotification")
                    .WithSimpleSchedule(x => x
                        .WithIntervalInMinutes(JobEngineReoccurrenceMinutes)
                        .RepeatForever())
                .Build();

                await _scheduler.ScheduleJob(job, trigger);
```

**Comment by lahma** [comment]
Simple trigger's first run is at start, you've given start now (immediately), try different start times.

**Comment by s3YwCf2ZbfJG4SHAfjQMAjtsf** [comment]
StartNow is commented out.   But it still runs at creation.   I tried WithCalendarIntervalSchedule but that didn't seem to work.

It was a minor problem, so I put a hack boolean in play to prevent it from running at creation.

**Reply by lahma** [reply]
I think `DateTime.UtcNow` is the default.
